README.md

```diff
@@ -139,7 +139,7 @@ yum install qemu libvirt libvirt-devel ruby-devel gcc qemu-kvm
 
 * Fedora 22 and up:
 ```shell
-dnf -y install qemu libvirt libvirt-devel ruby-devel gcc
+dnf install -y gcc libvirt libvirt-devel libxml2-devel make ruby-devel
```

Comment:
quick check of the rpm spec file suggests that libvirt in fedora depends on qemu, hence the dropping of it here.